docs/api2.yaml

```
@@ -283,9 +283,391 @@ paths:
+            format: password
+      responses:
+        "200":
+          description: Deleted
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    default: success
+                  data:
+                    default: null
+        "401":
+          description: Unauthorized (Invalid password)
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    default: fail
+                  data:
+                    type: string
+                    example:
+                      password: Invalid or missing password
+        "400":
+          description: Bad request (Tried to delete foreign comment)
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    default: fail
+                  data:
+                    type: string
+                    example:
+                      userID: You can only delete your own comments
+        "403":
+          $ref: "#/components/responses/UnsupportedClientVersion"
+        "404":
+          description: Crowdaction or comment not found
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    default: fail
+                  data:
+                    type: string
+                    example:
+                      crowdactionID: crowdaction does not exist
+      security:
+        - FirebaseAuthorizer: []
+  "/crowdactions/{crowdactionID}/comments/{commentID}/report":
+    post:
+      tags:
+        - Crowdaction
+      summary: Report a specific comment on a specific crowdaction
+      parameters:
+        - $ref: "#/components/parameters/ApiVersionParameter"
+        - name: crowdactionID
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: commentID
+          in: path
+          required: true
+          schema:
+            type: string
+        - in: query
+          name: password
+          description: Only include if crowdaction requires password
+          required: false
+          schema:
+            type: string
+            format: password
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                reason:
+                  type: string
+                  enum:
+                    - spam
+                    - hatespeach
+                    - falsehood
+                    - doxing
```

```suggestion
                  enum:
                    - discrimination
                    - doxing
                    - falsehood
                    - hatespeech
                    - racism
                    - spam
```
Also for this one `spam`, let's maybe add some kind of block if a person is making more than # amount of comments per # amount minutes? Opinion?

---

I guess that will be another 400 Bad request (Too many requests) maybe?
But this is probably an implementation detail.  
I'll document it in the issue.